### ISSUE TITLE
Ideal way of storing data for access tokens/ refresh tokens

### ISSUE THREAD

**Issue by DillonSimeone** [Shiroraven/Jikanri#9]
Storing sensitive user information in localstorage is something we have to avoid at all cost. Due to localstorage being readable and unencrypted.

_Originally posted by @Shiroraven in https://github.com/Shiroraven/Jikanri/pull/8#issuecomment-602430699_

You talked about Svelte.store when we touched onto this in Discord... After staring at the docs and tutorials for a bit, then looking around, I stumbled across those:

https://stackoverflow.com/questions/56488202/how-to-persist-svelte-store

https://www.hacktension.com/question/WmhZWERpM01mekt3V016c1c3ZksrQT09/how-can-i-persist-sveltes-store-like-redux-does

Tl;Dr, if the user refreshes the page, all svelte.store data will be lost. 

To work with this, we could store just enough infomation in localStorage to silently trigger the signIn function for the providers the users have logged in if the user refreshes the page for any reason.

**Comment by Shiroraven** [comment]
Alright, my bad! Guess localstorage or cookies are the only 2 viable options without using a database.

**Comment by Shiroraven** [comment]
The ideal way would certainly be storing all this data away from the frontend using a firestore database not sure what the usage limits are on those though.

**Comment by DillonSimeone** [comment]
I've confirmed that svelte store data does not persist across reloads. 

Sign in... The svelte writable store is filled up with all needed data.

![image](https://user-images.githubusercontent.com/21962421/77407788-ee020d80-6d73-11ea-84bd-78fad55fd948.png)

Refresh the page, and you see that the svelte store object is now empty.


![image](https://user-images.githubusercontent.com/21962421/77407700-cdd24e80-6d73-11ea-853d-4195d2e2a43e.png)

Sad!

**Comment by DillonSimeone** [comment]
Yeah. We've gotta to do some reading on that.

https://blog.logrocket.com/jwt-authentication-best-practices/

I was talking about HTTPonly cookies this morning... But it appears that those cookies require some backend code to run to be useable. 

Hmmm...

https://dev.to/realabbas/how-to-secure-jwt-token-in-react-172a

Googling `store JWT token` is returning all kinds of good results.
